docs/platforms/javascript/common/configuration/integrations/launchdarkly.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: LaunchDarkly
+description: "Attaches recent LD feature flag evaluations to error event context."
+notSupported:
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.bun
+  - javascript.capacitor
+  - javascript.cloudflare
+  - javascript.connect
+  - javascript.cordova
+  - javascript.deno
+  - javascript.electron
+  - javascript.express
+  - javascript.fastify
+  - javascript.gcp-functions
+  - javascript.hapi
+  - javascript.koa
+  - javascript.nestjs
+  - javascript.nodejs
+  - javascript.wasm
```

Comment:
These are the non-frontend frameworks

Reply:
feature flags can be used on BE too. Does LD for example have a packge for browser and another for node?

Reply:
LD has server-side node, yes, but the implementation + installation will look different, so we focus on client-side for now. See https://github.com/getsentry/sentry-javascript/pull/14207#issuecomment-2471504674

Reply:
Tldr; the hook used to listen for evaluations doesn't exist for node. If it does, it works differently. This is also the reason we can't support specific frontend frameworks, only the generic launchdarkly-js-client-sdk